StripePayments/StripePayments/Source/PaymentHandler/STPPaymentHandler.swift

```diff
@@ -1604,11 +1621,19 @@ public class STPPaymentHandler: NSObject {
     /// 1. Redirects to an app using url
     /// 2. Open fallbackURL in a webview if 1) fails
             ///
-    func _handleRedirect(to nativeURL: URL?, fallbackURL: URL?, return returnURL: URL?) {
+    func _handleRedirect(to nativeURL: URL?, fallbackURL: URL?, return returnURL: URL?, completion: ((SFSafariViewController?) -> Void)? = nil) {
```

Comment:
Can you add a docstring explaining `completion`?

Reply:
Added!